### PR TITLE
Readme update

### LINKED ISSUE #1: Commands are lost in some cases
Lets describe case. If you are going in up direction and press right and up arrow fast, up arrow press is ignored.

### PR DESCRIPTION
**Pull request by djolewalker**

### CODE REVIEW

**Review by DejanBesic — CHANGES_REQUESTED** [review]
Change this!

**Review by djolewalker — COMMENTED** [review]
Done

**Review by DejanBesic — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 Student project. Advanced arrays usage. Plain JS. Custom snake algorithm.
 
+## Live
+
```
Comment (djolewalker) [inline README.md:6]: Remove this line

Comment (djolewalker) [inline README.md:6]: simpleee

**Review by djolewalker — COMMENTED** [review]
sss

File: README.md (hunk 1 above)
Comment (djolewalker) [reply README.md:6]: asdad

**Pull request closed without merging**